src/components/frame/v5/pages/FundsPage/partials/FundsTable/hooks.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { useMemo, useState } from 'react';
+import { ColumnDef, createColumnHelper } from '@tanstack/react-table';
+
+import TokenTable from '../TokenTable';
+import {
+  FundsTableFilters,
+  FundsTableModel,
+  UseFundsTableProps,
+} from './types';
+import { useColonyContext, useColonyFundsClaims } from '~hooks';
+import { notNull } from '~utils/arrays';
+import { FilterProps } from '../Filter/types';
+import TokenIcon from '~shared/TokenIcon';
+import { formatText } from '~utils/intl';
+
+export const useFundsTableColumns = (): ColumnDef<
+  FundsTableModel,
+  string
+>[] => {
+  const columnHelper = useMemo(() => createColumnHelper<FundsTableModel>(), []);
+
+  const columns: ColumnDef<FundsTableModel, string>[] = useMemo(
+    () => [
+      columnHelper.display({
+        header: () => null,
+        id: 'token',
+        cell: ({ row }) => (
+          <TokenTable
+            key={row.original.token?.tokenAddress}
```

Comment:
Is key needed here?

Reply:
It is needed to close accordions while filtering/sorting